tests/test_samplings.py

```diff
@@ -492,3 +439,49 @@ def test_em64():
 
     npt.assert_allclose(
         np.sum(sampling.weights), 4*np.pi, atol=1e-6, rtol=1e-6)
+
+
+def test_hyperinterpolation(download_sampling):
+    n_max = 1
+    download_sampling('hyperinterpolation', n_max)
```

Comment:
Is this line required? The samping should also be downloaded in the function call in the next line.